device/mellanox/x86_64-mlnx_msn3420-r0/ACS-MSN3420/port_config.ini

```diff
@@ -0,0 +1,61 @@
+Ethernet0       0                   etp1      0
+Ethernet4       4                   etp2      1
+Ethernet8       8                   etp3      2
+Ethernet12      12                  etp4      3
+Ethernet16      16                  etp5      4
+Ethernet20      20                  etp6      5
+Ethernet24      24                  etp7      6
+Ethernet28      28                  etp8      7
+Ethernet32      32                  etp9      8
+Ethernet36      36                  etp10     9
+Ethernet40      40                  etp11     10
+Ethernet44      44                  etp12     11
+Ethernet48      48                  etp13     12
+Ethernet52      52                  etp14     13
+Ethernet56      56                  etp15     14
+Ethernet60      60                  etp16     15
+Ethernet64      64                  etp17     16
+Ethernet68      68                  etp18     17
+Ethernet72      72                  etp19     18
+Ethernet76      76                  etp20     19
+Ethernet80      80                  etp21     20
+Ethernet84      84                  etp22     21
+Ethernet88      88                  etp23     22
+Ethernet92      92                  etp24     23
+Ethernet96      96                  etp25     24
+Ethernet100     100                 etp26     25
+Ethernet104     104                 etp27     26
+Ethernet108     108                 etp28     27
+Ethernet112     112                 etp29     28
+Ethernet116     116                 etp30     29
+Ethernet120     120                 etp31     30
+Ethernet124     124                 etp32     31
+Ethernet128     128                 etp33     32
+Ethernet132     132                 etp34     33
+Ethernet136     136                 etp35     34
+Ethernet140     140                 etp36     35
+Ethernet144     144                 etp37     36
+Ethernet148     148                 etp38     37
+Ethernet152     152                 etp39     38
+Ethernet156     156                 etp40     39
+Ethernet160     160                 etp41     40
+Ethernet164     164                 etp42     41
+Ethernet168     168                 etp43     42
+Ethernet172     172                 etp44     43
+Ethernet176     176                 etp45     44
+Ethernet180     180                 etp46     45
+Ethernet184     184                 etp47     46
+Ethernet188     188                 etp48     47
+Ethernet192     192,193,194,195     etp49     48
+Ethernet196     196,197,198,199     etp50     49
+Ethernet200     200,201,202,203     etp51     50
+Ethernet204     204,205,206,207     etp52     51
+Ethernet208     208,209,210,211     etp53     52
+Ethernet212     212,213,214,215     etp54     53
+Ethernet216     216,217,218,219     etp55     54
+Ethernet220     220,221,222,223     etp56     55
+Ethernet224     224,225,226,227     etp57     56
+Ethernet228     228,229,230,231     etp58     57
+Ethernet232     232,233,234,235     etp59     58
+Ethernet236     236,237,238,239     etp60     59
```

Comment:
should you have new line at the end of the file or not? not sure about it just double checking

Reply:
fixed